pkg/operation/cloudbotanist/alicloudbotanist/controlplane.go

```diff
@@ -135,6 +136,12 @@ func (b *AlicloudBotanist) GenerateCSIConfig() (map[string]interface{}, error) {
 	)
 }
 
+// MetadataServiceAddress returns Aliyun's MetadataService address.
+func (b *AlicloudBotanist) MetadataServiceAddress() (*net.IPNet, error) {
+	_, cidr, err := net.ParseCIDR("100.100.100.200/32")
+	return cidr, err
```

Comment:
The  function does not really need to return an error - the input for `net.ParseCIDR` is static/hard-coded and not controlled from outside.